Content.Shared/Corvax/Documents/DocumentPrinterComponent.cs

```diff
@@ -0,0 +1,10 @@
+using Content.Shared.Containers.ItemSlots;
+
+namespace Content.Shared.Corvax.Documents;
+
+[RegisterComponent]
+public sealed partial class DocumentPrinterComponent : Component
+{
+    [DataField]
+    public ItemSlot IdSlot = new();
+}
```

Comment:
```suggestion
[RegisterComponent]
public sealed partial class DocumentPrinterComponent : Component;
```
Или, чтобы не хардкодить название слота в системе можно:
```suggestion
[RegisterComponent]
public sealed partial class DocumentPrinterComponent : Component
{
    [DataField]
    public string SlotName = "Id";
}
```